File: 0001_Multiples of 3 and 5/0001.ts
function findMultiples(max: number){
    let count: number;
    count = 0;

    let i: number;
    for (i = 0; i<=max; i+=3) {
        count+=i;
    }
    for (i = 0; i<=max; i+=5) {
        count+=i;
    }

    console.log(count);
}

findMultiples(1000);

File: 0001_Multiples of 3 and 5/0001.c
#include <stdio.h>

void findMultiples(int max) {
    int count = 0;
    int i;
    for (i=0; i<max; i+=3) {
        if (!(i%5 == 0)) {
            count+=i;
        }
    }
    i = 0;
    for (i=0; i<max; i+=5) {
        count+=i;
    }
    printf("%i\n", count);
}

void main() {
    int input = 1000;
    findMultiples(input);
}
